### PR TITLE
background fetch; data_unpackaged; fix metric parser

### PR DESCRIPTION
**Pull request by hlolli**
@vird's changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +49,12 @@
     "electron-builder": "^24.6.4",
     "eslint": "^8.52.0",
     "eslint-plugin-react": "^7.33.2",
+    "husky": "^8.0.3",
+    "lint-staged": "^15.0.2",
     "next": "^12.3.4",
     "nextron": "^8.12.0",
     "postcss": "^8.4.31",
+    "prettier": "^3.0.3",
```
Comment (hlolli) [inline package.json:58]: please place it in devDependenceis

File: package.json [hunk 2]
```diff
@@ -32,9 +49,12 @@
     "electron-builder": "^24.6.4",
     "eslint": "^8.52.0",
     "eslint-plugin-react": "^7.33.2",
+    "husky": "^8.0.3",
+    "lint-staged": "^15.0.2",
```
Comment (hlolli) [inline package.json:54]: please place it in devDependenceis

File: package.json [hunk 3]
```diff
@@ -32,9 +49,12 @@
     "electron-builder": "^24.6.4",
     "eslint": "^8.52.0",
     "eslint-plugin-react": "^7.33.2",
+    "husky": "^8.0.3",
```
Comment (hlolli) [inline package.json:53]: please place it in devDependenceis

File: package.json (hunk 1 above)
Comment (mrsaifullah52) [reply package.json:58]: yes its under devDependencies
image

**Pull request merged**